src/txn/txn_ckpt.c

```diff
@@ -1151,7 +1152,16 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     tsp.tv_sec = 10;
     tsp.tv_nsec = 0;
     __checkpoint_timing_stress(session, WT_TIMING_STRESS_CHECKPOINT_SLOW, &tsp);
+
+    WT_STAT_CONN_SET(session, checkpoint_state, WT_CHECKPOINT_STATE_CKPT_TREE);
+
+    time_start_ckpt_tree = __wt_clock(session);
     WT_ERR(__checkpoint_apply_to_dhandles(session, cfg, __checkpoint_tree_helper));
+    time_stop_ckpt_tree = __wt_clock(session);
+    ckpt_tree_duration_usecs = WT_CLOCKDIFF_US(time_stop_ckpt_tree, time_start_ckpt_tree);
+    WT_STAT_CONN_SET(session, txn_ckpt_tree_duration, ckpt_tree_duration_usecs);
+
+    __checkpoint_verbose_track(session, "transaction checkpoint tree");
```

Comment:
This should match the tense/tone of the other `__checkpoint_verbose_track` calls. Those calls are all done before the expensive work, so I would suggest moving it ahead of the `__checkpoint_apply_to_dhandles` call and changing the text to "checkpointing individual trees".

Reply:
hi, @korteland 
Thank you for your patient guidance， I have change the code again.
thanks, please review again.